database/seed-data/fakeDataGenerator.js

```diff
@@ -31,6 +31,11 @@ const fakeAnswers = async () => {
 };
 
 const fakeDataGenerator = async (recordsToCreate, startingRecordID) => {
+
+  if (recordsToCreate === undefined || startingRecordID === undefined) {
```

Comment:
Nice error check for if an invalid parameter is put into your fakeDataGenerator function.